apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.Data/SpannerParameter.cs

```diff
@@ -86,108 +59,96 @@ public SpannerParameter(string spannerColumnName, object value) : this(spannerCo
         public SpannerParameter(string spannerColumnName, SpannerDbType type, string sourceColumn) : this(
             spannerColumnName, type)
         {
-            throw new NotImplementedException();
+            _sourceColumn = sourceColumn;
         }
 
         /// <inheritdoc />
         public override DbType DbType
         {
-            get { throw new NotImplementedException(); }
-            set { throw new NotImplementedException(); }
+            get { return _spannerTypeCode.GetDbType(); }
+            set { _spannerTypeCode = value.GetSpannerType(); }
         }
 
         /// <inheritdoc />
         public override ParameterDirection Direction
         {
-            get { throw new NotImplementedException(); }
-            set { throw new NotImplementedException(); }
+            get { return _direction; }
+            set
+            {
+                throw new InvalidOperationException("Spanner does not support anything except input parameters.");
```

Comment:
"Spanner supports input parameters only ."

I think starting a sentence with "Spanner does not support anything"  reads poorly.